local_pathfinding/objectives.py

```diff
@@ -64,14 +104,21 @@ class DistanceObjective(Objective):
             Only defined if the method is latlon.
     """
 
-    def __init__(self, space_information, method: DistanceMethod, reference=cs.LatLon(0, 0)):
-        super().__init__(space_information)
+    def __init__(
+        self,
+        space_information,
+        method: DistanceMethod,
+        reference: cs.LatLon = cs.LatLon(0, 0),
+        num_samples: int = 100,
```

Comment:
If you don't think `num_samples` won't change dynamically, we can remove this argument and just hardcode when calling the super's init function

```python
super().__init__(space_information, num_samples=100)
```

Reply:
Yup, num_sample won't change dynamically but I think it will change for each objective function. A more complicated objective function would require more samples to find a close estimate of the max_motionCost. Sure, we can hardcode it.